package.go

```diff
@@ -68,14 +68,29 @@ func (pkg *Package) GeneratePackage(dir string) error {
 	if err := pkg.generateFile("package", dir); err != nil {
 		return err
 	}
-	if err := pkg.generateFile("conversions", dir); err != nil {
+	if err := pkg.generateFile("package_notwindows", dir); err != nil {
```

Comment:
Having all these files is super unfortunate, but I'm not sure whether there is anything to be done for it.

Reply:
Done (created the common conversion.tmpl)